qulice-findbugs/src/main/java/com/qulice/findbugs/Wrap.java

```diff
@@ -76,6 +75,13 @@ public static void main(final String... args) {
     /**
      * Run it.
      * @param args Arguments
+     * @todo #706:30min After fix issue mentioned in comment
+     *  https://github.com/teamed/qulice/issues/706#issuecomment-225943946
+     *  rollback the changes done in #706 then remove @Ignore above
+     *  FindBugsValidatorTest.reportsIncorrectlyAddedThrows test
+     *  and fix test findbugs-bed-bogus-exception-declaration.
+     *  For the moment third item is done from comment
```

Comment:
@dskalenko why do we need information about the third item?  Will it in any way help solving this puzzle?